src/replication.c

```diff
@@ -2335,9 +2335,9 @@ void readSyncBulkPayload(connection *conn) {
 
     /* Final setup of the connected replica <- primary link */
     if (conn == server.repl_rdb_transfer_s) {
-        /* In case of full-sync using rdb channel, the primary client was already created for psync purposes
+        /* In case of full-sync using rdb connection, the primary client was already created for psync purposes
          * Instead of creating a new client we will use the one created for partial sync */
-        completeTaskRDBChannelSyncRdbConn(conn);
+        completeTaskRDBConnectionSyncRdbConn(conn);
```

Comment:
This name now contains "connection" and "conn".

Reply:
We can rename it to `rdbConnectionSyncRdbLoaded`, so the main connection finish function would be
`rdbConnectionSyncPsyncEstablished` instead of `completeTaskRDBConnectionSyncMainConn`.

Reply:
Yes these names look better. Thanks.